runtime/src/xor_fee_impls.rs

```diff
@@ -33,6 +33,7 @@ use crate::*;
 use bridge_types::{traits::EVMBridgeWithdrawFee, GenericNetworkId};
 #[cfg(feature = "wip")] // Dynamic fee
 use common::prelude::FixedWrapper;
+use common::weights::constants::MINIMAL_FEE;
```

Comment:
To be consistent, it is better to add it into `use common::prelude::constants::{BIG_FEE, SMALL_FEE};`, line 63 in `lib.rs`